tests/ui-focused/analysis-answers-before.ref

```diff
@@ -1,3 +1,3 @@
 {
-  "unusedstructures" : "Not answered"
+  "unusedstructures" : "{\n  \"answerElements\" : [\n    {\n      \"class\" : \"org.batfish.datamodel.answers.StringAnswerElement\",\n      \"answer\" : \"Not answered\"\n    }\n  ],\n  \"status\" : \"NOTFOUND\"\n}"
```

Comment:
the clients are currently written to expect strings, so we could embed arbitrary messages. not  the best decision in hindsight; the whole interface between clients and coordinator is a little hacky (e.g., no use of classes and packing responses by hand), which we need to fix at some point.